src/transformers/tokenization_utils_base.py

```diff
@@ -1965,24 +1965,37 @@ def convert_added_tokens(obj: Union[AddedToken, Any]):
             # Sort added tokens by index
             added_tok_encoder_sorted = list(sorted(added_tok_encoder.items(), key=lambda x: x[1]))
 
+            # Accumulate added tokens into batches of special/non-special tokens
```

Comment:
I think we can expend a little more this comment to clarify why we do this (to avoid rebuilding the slow tokenizer sort too often). :smile:

Reply:
Thank you. I extended the explanation in 625fcfcbd.